run_blockchain_tests.sh

```diff
@@ -48,21 +45,23 @@ case "$blockchain" in
     esac
 
 # taken from https://github.com/bitcoindevkit/bitcoin-regtest-box
-export BDK_RPC_AUTH=USER_PASS
-export BDK_RPC_USER=admin
-export BDK_RPC_PASS=passw
+export BDK_RPC_AUTH=COOKIEFILE
+export BDK_RPC_COOKIEFILE=/tmp/regtest.cookie
 export BDK_RPC_URL=127.0.0.1:18443
 export BDK_RPC_WALLET=bdk-test
 export BDK_ELECTRUM_URL=tcp://127.0.0.1:60401
 
 cli(){
-    docker exec -it "$id" /root/bitcoin-cli -regtest -rpcuser=admin -rpcpassword=passw $@
+    docker exec -it test_container /root/bitcoin-cli -regtest -datadir=/root/.bitcoin $@
```

Comment:
I'd prefer to keep this as `$id` and not use container names so different runs don't conflict.

Reply:
As long as the container is killed there won't be any data kept from a prior run, that's why I added the `docker --rm` option. I also tested with consecutive starting, generating blocks, killing the `test_container` and confirmed that the data is always reset back to a fresh state.

Reply:
I think it still makes sense to have unique ids so that tests can be run in parallel maybe? Like multiple blockchains at the same time

Reply:
sorry yes this is what I meant.

Reply:
The problem we will have with parallel containers is we'll also need unique ports assigned to each container. Which may be hard to do now but will be easier with the test framework like [electrsd](https://github.com/RCasatta/electrsd) that @RCasatta is working on. 

So I see your point, I'll switch this PR back to use `$id`. :slightly_smiling_face:

Reply:
Now that I think about it, if we switch to test frameworks that configuring and start the daemons that each test needs then we may not need docker containers at all.. anyway I'll still change it back.

Reply:
I think it's still gonna be useful to have docker containers to use them as an environment with everything pre-installed. Otherwise every time you have to download core, compile electrs, etc.

I think eventually we will probably manage to remove all of our bash scripts and just keep using the containers to ship binaries